packages/dataparcels/src/parcel/__test__/ModifyMethods-test.js

```diff
@@ -240,44 +240,90 @@ test('Parcel.modifyShapeUp() should modify value', () => {
 });
 
 test('Parcel.initialMeta() should work', () => {
-    expect.assertions(3);
+    let handleChange = jest.fn();
 
-    let meta = {a:1, b:2};
+    let parcel = new Parcel({
+        value: 123,
+        handleChange
+    }).initialMeta({a:1, b:2});
 
-    var data = {
+    expect(parcel.data).toEqual({
         value: 123,
-        handleChange: (parcel: Parcel) => {
-            let {meta} = parcel.data;
-            expect({a:1, b:3}).toEqual(meta);
-            expect({a:1, b:3}).toEqual(parcel.initialMeta().meta);
-        }
-    };
+        meta: {
+            a: 1,
+            b: 2
+        },
+        child: undefined,
+        key: "^"
+    });
 
-    let parcel = new Parcel(data).initialMeta(meta);
-    expect(meta).toEqual(parcel.meta);
-    parcel.setMeta({
-        b: 3
+    parcel.setMeta({b: 3});
+
+    expect(handleChange.mock.calls[0][0].data).toEqual({
+        value: 123,
+        meta: {
+            a: 1,
+            b: 3
+        },
+        child: undefined,
+        key: "^"
     });
 });
 
 test('Parcel.initialMeta() should merge', () => {
-    expect.assertions(2);
+    let handleChange = jest.fn();
 
-    let meta = {a:1, b:2};
-    let meta2 = {b:1, c:3}; // this b will be ignored because it will have already been set by the time this is applied
+    let parcel = new Parcel({
+        value: 123,
+        handleChange
+    })
+        .initialMeta({a:1, b:2})
+        .initialMeta({b:3, c:4})
 
-    var data = {
+    expect(parcel.data).toEqual({
         value: 123,
-        handleChange: (parcel: Parcel) => {
-            let {meta} = parcel.data;
-            expect({a:1, b:3, c:3}).toEqual(meta);
-        }
-    };
+        meta: {
+            a: 1,
+            b: 2,
+            c: 4
+        },
+        child: undefined,
+        key: "^"
+    });
+
+    parcel.setMeta({d: 5});
+
+    expect(handleChange.mock.calls[0][0].data).toEqual({
+        value: 123,
+        meta: {
+            a: 1,
+            b: 2,
+            c: 4,
+            d: 5
+        },
+        child: undefined,
+        key: "^"
+    });
+});
+
+test('Parcel.initialMeta() should do nothing to data if all meta keys are already set', () => {
+    let handleChange = jest.fn();
+
+    let parcel = new Parcel({
+        value: 123,
+        handleChange
+    })
+        .initialMeta({a:1, b:2})
+        .initialMeta({a:1, b:2}) // do nothing to data
```

Comment:
How does this test confirm that nothing is done to the data after the second `initialMeta` call?